test/specs/TC8_Sort_ZA.js

```diff
@@ -0,0 +1,19 @@
+import loginPage from '../pageobjects/login.page.js'
+import inventoryPage from '../pageobjects/inventory.page.js';
+import { config } from '../../config.js';
+
+const { username, password } = config.valid_user;
+
+describe('TC8 - Inventory Sort test ZA', async () => {
+    before(async () => {
+        await loginPage.open()
+        await loginPage.login(username, password)
+    });
+
+    it('should sort items by name from Z to A', async () => {
+        await inventoryPage.sort.selectByAttribute('value', 'za')
+        const itemNameTexts = await inventoryPage.allItems.map((item) => item.getText())
+        const sortedNames = [...itemNameTexts].sort((a, b) => b.localeCompare(a))
```

Comment:
@karenn32 How is that could be equal to the actual result? You reverse the sorting on the UI, it must be reversed for the ER as well. And the same thing about assertion for the TC7 goes here as well.

Reply:
Not actual.